### PR TITLE
Sreeraj/validation

### PR DESCRIPTION
**Pull request by djsreeraj**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,13 +61,13 @@
...
     "@types/snowpack-env": "2.3.4",
-    "typescript-eslint-parser": "^22.0.0",
     "autoprefixer": "10.4.0",
     "eslint-import-resolver-typescript": "^2.5.0",
     "eslint-plugin-import": "^2.25.2",
     "postcss": "8.3.11",
     "postcss-cli": "9.0.2",
     "snowpack": "3.8.8",
-    "typescript": "4.4.4"
+    "typescript": "4.4.4",
```
Comment (kirananto) [inline package.json:70]: class-validator missing in package json

File: src/Pages/Home/AddCustomerModal/InviteExisting.tsx [hunk 2]
```diff
@@ -68,41 +132,51 @@ export default function InviteExisting({
...
+		    </span>
       </div>
       {/* <!-- btn popup --> */}
       <div className="flex my-8 p-2 gap-2">
 
         <button onClick={() => setCurrentPage(currentPageEnum.STEP_1)} className="save-btn p-3 w-full text-indigo-700 rounded-full border border-indigo-700  dark:border-indigo-200 dark:text-indigo-200">
           Back
         </button>
-        <button onClick={() => setCurrentPage(currentPageEnum.PREVIEW)} className="save-btn p-3 w-full text-white rounded-full bg-gradient-to-br from-blue-500 to-indigo-700">
+        <button onClick={() =>  handleSubmit()} className="save-btn p-3 w-full text-white rounded-full bg-gradient-to-br from-blue-500 to-indigo-700">
```
Comment (kirananto) [inline src/Pages/Home/AddCustomerModal/InviteExisting.tsx]: You can provide direct function here

File: src/Pages/Home/AddCustomerModal/Step1.tsx [hunk 3]
```diff
@@ -14,6 +16,12 @@ interface IProps {
   setIsExisting: any
   isExisting: any
 }
+
+export class Post {
+  @Length(10, 10)
+  phoneNumber!: string;
```
Comment (kirananto) [inline src/Pages/Home/AddCustomerModal/Step1.tsx:22]: Let's make it 10-12 for safety

File: package.json (hunk 1 above)
Comment (djsreeraj) [reply package.json:70]: its in line 27

Comment (kirananto) [reply package.json:70]: Cool 👍

**Review by kirananto — APPROVED** [review]

**Pull request merged**